sound/hybrid_font.h

```diff
@@ -503,7 +506,7 @@ class HybridFont : public SaberBase, public Looper {
           PlayPolyphonic(&SFX_in);
 	  hum_fade_out_ = 0.2;
         }
-	check_postoff_ = !!SFX_pstoff;
+        if (off_type != OFF_FAST) check_postoff_ = !!SFX_pstoff;
```

Comment:
probably better to always set it:

check_postoff_ = !!SFX_pstoff && off_type != OFF_FAST;

Reply:
updated